google-maps-compose-sample/src/main/java/com/what3words/maps/google_maps_compose_sample/UsingMapFragmentActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.what3words.maps.google_maps_compose_sample
+
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.viewinterop.AndroidViewBinding
+import androidx.fragment.app.FragmentActivity
+import com.google.android.gms.maps.GoogleMap.MAP_TYPE_NORMAL
+import com.what3words.components.maps.models.W3WMarkerColor
+import com.what3words.components.maps.models.W3WZoomOption
+import com.what3words.components.maps.views.W3WGoogleMapFragment
+import com.what3words.components.maps.views.W3WMap
+import com.what3words.maps.google_maps_compose_sample.databinding.ActivityUsingMapFragmentBinding
+import com.what3words.maps.google_maps_compose_sample.ui.theme.W3wandroidcomponentsmapsTheme
+
+class UsingMapFragmentActivity : FragmentActivity(), W3WGoogleMapFragment.OnMapReadyCallback {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            W3wandroidcomponentsmapsTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colors.background
+                ) {
+                    AndroidViewBinding(
+                        { inflater, parent, attachToParent ->
```

Comment:
Since we specified the named params for the modifier, we might as well just specify the factory name param for the AndroidViewBinding composable